docs/index.rst

```diff
@@ -0,0 +1,35 @@
+.. geocat-viz documentation master file, created by
+   sphinx-quickstart on Tue Jun 22 13:21:57 2021.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+.. .. image:: _static/images/nsf.png
+..    :scale: 100%
+..   :align: right
+
+.. |
+.. |
+
+GeoCAT-viz
+==========
+
```

Comment:
Knowing that this docs fodler and index.rst will initiate GeoCAT-viz wevpage but entirely empty, could you at least put a paragraph of the descriptive text about GeoCAT-viz (probably from contributor's guide) here?

Reply:
That's fair! I stole a paragraph from the README for the moment.